ONSdigital/eq-author
# 449 - Earliest date validation

### What is the context of this PR?
This change implements updating earliest date validation for `Date` answers.

### How to review 
1. Add a Date answer to a page.
2. Ensure you can enable it and update all the fields in the modal that is opened.

### Dependencies
- [x] Schema - https://github.com/ONSdigital/eq-author-graphql-schema/pull/62
- [x] API - https://github.com/ONSdigital/eq-author-api/pull/119
